staging/src/k8s.io/apiserver/pkg/cel/library/cost.go

```diff
@@ -235,6 +235,23 @@ func (l *CostEstimator) CallCost(function, overloadId string, args []ref.Val, re
 		// url accessors
 		cost := uint64(1)
 		return &cost
+	case "_==_":
+		if len(args) == 2 {
+			unitCost := uint64(1)
+			lhs := args[0]
+			switch lhs.(type) {
+			case cel.Quantity:
+				return &unitCost
+			case cel.IP:
+				return &unitCost
+			case cel.CIDR:
+				return &unitCost
+			case *cel.Format: // Formats have a small max size.
+				return &unitCost
+			case cel.URL: // TODO: Computing the actual cost is expensive, and changing this would be a breaking change
+				return &unitCost
+			}
+		}
```

Comment:
These are the "runtime costs".  They default to 1.  I've made them explicit here to document them and provide framework for future types to compute appropriate costs.

Note that URL cost is wrong, it should be based on the length of the URLs.  Fixing this would be a breaking change and I won't do anything with it in this PR.

Reply:
I guess a unit test that set `panicOnUnknown` and automatically checked runtime and estimated cost for all the types defined in our function params / return values would have caught that this was undefined